include/mbedtls/config_psa.h

```diff
@@ -838,6 +838,7 @@ extern "C" {
 #define MBEDTLS_PSA_BUILTIN_KEY_TYPE_RSA_KEY_PAIR_IMPORT 1
 #define MBEDTLS_PSA_BUILTIN_KEY_TYPE_RSA_KEY_PAIR_EXPORT 1
 #define MBEDTLS_PSA_BUILTIN_KEY_TYPE_RSA_KEY_PAIR_GENERATE 1
+#define MBEDTLS_GENPRIME
```

Comment:
Hmm, it looks like this is the `!MBEDTLS_PSA_CRYPTO_CONFIG` section of the file that enables PSA symbols base on what's available in Mbed TLS, so I htink that should be
```c
#if defined(MBEDTLS_GENPRIME)
#define MBEDTLS_PSA_BUILTIN_KEY_TYPE_RSA_KEY_PAIR_GENERATE 1
#defined PSA_WANT_KEY_TYPE_RSA_KEY_PAIR_GENERATE 1
#endif
```
instead.

Then in the `MBEDTLS_PSA_CRYPTO_CONFIG` section of the file, where we enable what's needed in Mbed TLS based on what was asked on the PSA side, there indeed we should auto-enable `GENPRIME` whenever we enabled `BUILTIN_..._RSA_..._GENERATE`.